### DISCUSSION TITLE
Multiome vignette - error in TSSEnrichment

### DISCUSSION THREAD

**Discussion by sjbegg** [stuart-lab/signac#831] [Q&A] [answered]
Hello,

I am running the vignette on the 10XMultiome dataset, and hitting an error that I'm unsure how to resolve. 

```
> pbmc <- TSSEnrichment(pbmc)
Extracting TSS positions
Extracting fragments at TSSs
  |++++++++++++++++++++++++++++++++++++++++++++++++++| 100% elapsed=01s  

Computing TSS enrichment score
Error in ecdf(x = object$TSS.enrichment) : 
  'x' must have 1 or more non-missing values
```
Weirdly, this error didn't appear at the end of last week when I was running the code, and only when I started again this week. Any insight would be gratefully received as I'm totally stuck!

Thanks.

**Comment by timoast** [comment, marked as answer]
Answered here https://github.com/timoast/signac/issues/834
